### DISCUSSION TITLE
Can we change 9 images to 5 for evolve

### DISCUSSION THREAD

**Discussion by ImageAlchemy** [runew0lf/RuinedFooocus#154] [Q&A]
Im using the evolve fuction , it makes 9 prompts and makes 9 images  and auto renders. Is  there a way to modify it to 5 images ?

**Comment by yownas** [comment]
Not really. It is using the fact that you get a nice 3x3 grid in the preview to match the buttons. 4 (2x2) or 16 (4x4) might have worked, but 4 is to small and 16 felt like a lot.
